source/whats-new.txt

```diff
@@ -57,8 +57,38 @@ The 3.0 driver release includes the following new features:
   This type is available in .NET 5 and later.
   To learn more about the ``Half`` type, see the
   `Half Struct <https://learn.microsoft.com/en-us/dotnet/api/system.half?view=net-8.0>`__
-  API reference page on MSDN. 
-
+  API reference page on MSDN.
+
+- Adds support for implicit client-side projection when using the ``Find()`` and
```

Comment:
It won't be implicit.

Client-side projections are potentially undesirable and the current thinking is that they will have to "enable" client side projections in order to use them.

The current PR is not merged to master yet, but the two places client-side projections can be enabled are in the `MongoClientSettings` or in the `FindOptions` or `AggregateOptions` for an individual query.

Theoretically they could be implicit, but that's not the current thinking.

Reply:
Thanks for clarifying! I was trying to figure this out from the relevant ticket and PR but must have missed this aspect.